src/system.rs

```diff
@@ -67,4 +73,29 @@ pub fn sw_version() -> Result<serde_json::Value> {
         "osName": sw_versions[0],
         "swVersion": sw_versions[1],
     }))
-}
\ No newline at end of file
+}
+
+pub fn boot_time() -> Result<String> {
+    if cfg!(feature = "mock") {
+        Ok("2024-10-10T05:27:52.804875461Z".to_string())
+    } else {
```

Comment:
I'd probably vote for:

```rust
#[cfg(feature = "mock")]
pub fn boot_time() -> Result<String> {
...
}

#[cfg(not(feature = "mock"))]
pub fn boot_time() -> Result<String> {
...
}
```

instead to reduce function complexity. This has the disadvantage that not all branches are compiled with every build, but since its about the tests and we run the tests as part of the CI, I think this disadvantage is acceptable.